README.md

```diff
@@ -314,6 +314,49 @@ overrides the value specified in **Workspace**.
 | `independent` | `bool` | No | Yes | `version`, `publish` |
 | `no_individual_tags` | `bool` | Yes | No | `version`, `publish` |
 
+## Running tests
+
+The recommended way to run tests for the crate is as follows:
+
+```sh
+cargo test --manifest-path cargo-workspaces/Cargo.toml -j1
+```
+
+The integration tests manipulate the file system, so running them from multiple threads
+may cause race conditions and unexpected failures.
+
+### Adding tests and updating snapshots
+
+`cargo-workspaces` uses [`insta`](https://docs.rs/insta/) for snapshot testing.
+
+When updating tests, you may run tests via:
+
+```
+INSTA_UPDATE=always cargo test <args>
+```
+
+If you don't want to override any existing snapshots, use:
+
+```
+INSTA_UPDATE=unseen cargo test <args>
+```
+
+Always make sure that generated snapshots match what you expect the test to produce.
+Do not blindly commit changes in snapshots that you do not anticipate.
+
+For more details, check the `insta` documentation.
+
+### Troubleshooting
+
+If you observe unexpected differences in snapshots, you may want to override your compiler to
+the same version as used in [CI](.github/workflows/ci.yml), e.g.:
```

Comment:
CI is fixed on 1.70, and newer cargo doesn't emit `# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html` line.
Also, it looks like `cargo install --path cargo-workspaces` doesn't work with 1.70, so when I worked on it, I had to manually override compiler for tests and remove an override for installation.
I guess MSRV and snapshots should be updated, but I thought it'd be an unnecessary scope extension for this PR.

I believe that this comment is useful as a general observation regardless of that.